doc/man3/SSL_CTX_set_psk_client_callback.pod

```diff
@@ -129,7 +129,8 @@ B<SSL_psk_client_cb_func> callback.
 
 =head1 NOTES
 
-Note that parameter B<hint> given to the callback may be B<NULL>.
+Note that parameter B<hint> given to the callback may be B<NULL>, a default
+"Client_identity" string is used then instead.
```

Comment:
IMO this is wrong place - you're changing s_client and that's where the documentation should be amended.